apps/designer-standalone/src/app/LocalDesigner/pseudoCommandBar.tsx

```diff
@@ -51,6 +57,7 @@ export const PseudoCommandBar = () => {
         onClick={() => dispatch(switchToWorkflowParameters())}
       />
       <ActionButton iconProps={{ iconName: 'Code' }} text="Code View" onClick={serializeCallback} />
+      <ActionButton iconProps={{ iconName: 'ErrorBadge' }} text="Errors" onClick={() => dispatch(switchToErrorsPanel())} />
```

Comment:
Yes, we should be, though I'm not sure this is the right way to do it in all places, in portal we want to surface this as a panel likely. In VSCode it might be good to find a way to surface this in the Issues Tab